ui/components/ui/button/buttons.scss

```diff
@@ -101,7 +93,7 @@
 }
 
 .btn-danger-primary {
-  color: var(--color-background-default);
+  color: var(--color-error-inverse);
```

Comment:
The `inverse` category should be used for any foreground elements that go on top of the `default` category all the below text examples use the "inverse" in its respective category. 

![inverse](https://user-images.githubusercontent.com/8112138/155425495-cc52b514-2039-4dc1-8983-d5083bc77c5f.jpg)